grakn-kb/src/main/java/ai/grakn/kb/internal/ValidateGlobalRules.java

```diff
@@ -388,4 +388,15 @@ private static ReasonerQuery combinedRuleQuery(GraknTx graph, Rule rule){
 
         return errors;
     }
+
+    /**
+     * Checks if a {@link Relationship} has at least one role player.
+     * @param relationship The {@link Relationship} to check
+     */
+    static Optional<String> validateRelationshipHasRolePlayers(Relationship relationship) {
```

Comment:
Out of curiosity, is it ok to have a relationship with, let's say, 3 roles of which only 1 has a role player?

Reply:
Yes. You can have a single role-roleplayer mapping for a given relation instance plus you can also have a single roleplayer playing multiple roles.